commit f84b7eda91d997c2828b9fa7b7ba77db5a4fa680 (refs/changes/25/2025/2)
Author: Joshua M. Clulow <jmc@joyent.com>
Date:   2017-06-01T21:09:05+00:00 (2 years, 4 months ago)
    
    OS-6159 root.password file contains the hash, not the password
    Reviewed by: Cody Peter Mello <cody.mello@joyent.com>
    Approved by: Cody Peter Mello <cody.mello@joyent.com>

diff --git a/tools/build_live b/tools/build_live
index 97bf6ee9..cca886a8 100755
--- a/tools/build_live
+++ b/tools/build_live
@@ -375,7 +375,7 @@ function bi_create_password
 	# Stash the generated root password as a plain text build artefact
 	# so that we can find it later if we need it.
 	#
-	if ! printf '%s\n' "$cp_hash" >"$stash"; then
+	if ! printf '%s\n' "$cp_password" >"$stash"; then
 		fail "unable to stash root password in \"$stash\""
 	fi
 
